monitoring/metrics/worker_metrics.py

```
@@ -0,0 +1,108 @@
+
+def update_agent_activity(agent_id: str, agent_type: str, active: bool):
+    """Update agent activity status."""
+    AGENT_ACTIVITY.labels(agent_id=agent_id, agent_type=agent_type).set(1 if active else 0)
+
+def record_worker_error(error_type: str, agent_type: str):
+    """Record a worker error."""
+    WORKER_ERRORS.labels(error_type=error_type, agent_type=agent_type).inc()
+
+def metrics_endpoint():
+    """Return Prometheus metrics."""
+    return Response(generate_latest(), media_type=CONTENT_TYPE_LATEST)
```

Ruff (0.12.2)

Undefined name `Response`

(F821)